source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-table/table/header/sort-arrows.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import DownArrowIcon from '@material-ui/icons/ArrowDownward';
+import UpArrowIcon from '@material-ui/icons/ArrowUpward';
+
+
+const iconStyle = {
```

Comment:
is there a reason why this is not in a scss file? just for consistency

Reply:
not really. but I want to get this merged ASAP, so it can get pulled out later